core/src/main/scala/com/nvidia/spark/rapids/tool/planparser/DataWritingCommandExecParser.scala

```diff
@@ -28,16 +28,111 @@ case class DataWritingCommandExecParser(
   // hardcode because InsertIntoHadoopFsRelationCommand uses this same exec
   // and InsertIntoHadoopFsRelationCommand doesn't have an entry in the
   // supported execs file
-  val fullExecName = "DataWritingCommandExec"
+  val fullExecName = DataWritingCommandExecParser.dataWriteCMD
 
   override def parse: ExecInfo = {
-    val writeFormat = checker.getWriteFormatString(node.desc)
-    val writeSupported = checker.isWriteFormatsupported(writeFormat)
+    // At this point we are sure that the wrapper is defined
+    val wStub = DataWritingCommandExecParser.getWriteCMDWrapper(node).get
+    val writeSupported = checker.isWriteFormatSupported(wStub.dataFormat)
     val duration = None
-    val speedupFactor = checker.getSpeedupFactor(fullExecName)
+    val speedupFactor = checker.getSpeedupFactor(wStub.mappedExec)
```

Comment:
To confirm that with this change, we'll go from only having a speedup factor for `DataWritingCommandExec` to now having a speedup factor for `DataWritingCommandExec`, `SaveIntoDataSourceCommand`, and `InsertIntoHadoopFsRelationCommand`.  Is that correct?

Reply:
This change maps all the writeOp to `DataWritingCommandExec`.
Once we have the correct speedups for  `InsertIntoHadoopFsRelationCommand` and `SaveIntoDataSourceCommand`, the hashMap will be updated to load the new entry.